helm/alfresco-content-services/charts/alfresco-elasticsearch-connector/templates/reindexing-job.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.reindexing.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "alfresco-elasticsearch-connector.fullname" . }}-reindexing
```

Comment:
Now we are using the single-node version but in future we have to support the multi-node version https://alfresco.atlassian.net/browse/SEARCH-3075